src/app/features/rpc-transaction-request/switch-account-trigger/switch-account-trigger.tsx

```diff
@@ -32,26 +35,26 @@ export function TransactionSwitchAccount({ toggleSwitchAccount }: TransactionSwi
 
   const titleRight = (
     <SkeletonLoader isLoading={isLoadingBalance} width="96px">
-      <styled.span textStyle="label.02">{formatMoneyPadded(balance.availableBalance)}</styled.span>
+      <styled.span textStyle="label.02">{formatMoney(availableBalance)}</styled.span>
```

Comment:
IMO here we should use the padded amount

Reply:
I am always confused on this bc we lean toward it in code, but designs never show it. So, for both Bitcoin and Stacks showing the padded format is better? I think in design review they've asked me to change so I get confused about what is best.